README.md

```diff
@@ -1157,7 +1157,7 @@ The available fields are:
  - `playlist_title` (string): Playlist title
  - `playlist_uploader` (string): Full name of the playlist uploader
  - `playlist_uploader_id` (string): Nickname or id of the playlist uploader
- - `webpage_url` (string): A URL to the video webpage which if given to yt-dlp should allow to get the same result again
+ - `webpage_url` (string): A URL to the video webpage which if given to yt-dlp should allow one to get the same result again
```

Comment:
not sure about this. It is meant to say it allows yt-dlp to get the same result. No user involvement is implied

Reply:
You're right, in order to not duplicate 'it' so close together, I went with 'should allow yt-dlp' now.